Loop/Managers/Alerts/UserNotificationAlertScheduler.swift

```diff
@@ -65,7 +65,7 @@ fileprivate extension Alert {
         let userNotificationContent = UNMutableNotificationContent()
         userNotificationContent.title = backgroundContent.title
         userNotificationContent.body = backgroundContent.body
-        userNotificationContent.sound = userNotificationSound(muted: muted)
+        //userNotificationContent.sound = userNotificationSound(muted: muted)
```

Comment:
Is this intentional? If so, what's it about?

Reply:
thanks. not intentional